### PR TITLE
chore: update zh-Hant.json

### PR DESCRIPTION
**Pull request by den13501**
follow these commits to complete zhTW translation: 96021e518a7fa193576cd509d169c2a9f91f1c3c
8628d1e4b239c2852356f429d3ddbabbc5b35cc4
93d608f0504e5065ecb666601cdd3a58f52742c9
ce64894abe7f691e1db5cf3f9c03d0c93e927fa0
845297ec032e74a11a770a0ed9308d320f07c32f

### CODE REVIEW

File: web/src/locales/zh-Hant.json [hunk 1]
```diff
@@ -185,6 +192,8 @@
       "editor-font-style": "編輯器字體樣式",
       "mobile-editor-style": "行動版編輯器樣式",
       "default-memo-sort-option": "Memo 顯示時間",
+      "telegram-user-id": "Telegram UserID",
+      "telegram-user-id-placeholder": "向Telegram機器人傳送任意訊息以獲取",
```
Comment (Zeng1998) [inline web/src/locales/zh-Hant.json:196]:
```suggestion
      "telegram-user-id-placeholder": "向 Telegram 機器人傳送任意訊息以獲取",      
```

**Review by Zeng1998 — CHANGES_REQUESTED** [review]

**Review by Zeng1998 — APPROVED** [review]
LGTM.

**Comment by boojack** [comment]
Please fix the json format error

**Comment by athurg** [comment]
@den13501 I have fixed the JSON format error for you.

You can valid the JSON format before you commit a change and create PR next time: 

:)

**Review by boojack — APPROVED** [review]
LGTM, thanks for your contribution!

**Comment by den13501** [comment]
> e JSON format before you commit a change and create PR next time:

Thank you very much, I will be more careful next time!

**Pull request merged**